aarch64-blueprint.template

```diff
@@ -0,0 +1,21 @@
+name = "${IMAGE_NAME}"
```

Comment:
what if you specify these packages explicitly for x86 as well? if it works I think it's worth having one blueprint with all dependencies explicitly.